core/mutex.c

```diff
@@ -35,26 +35,27 @@
 #define ENABLE_DEBUG    (0)
 #include "debug.h"
 
-#define MUTEX_LOCKED ((void*)-1)
+#define MUTEX_LOCKED ((void *)-1)
 
 int _mutex_lock(mutex_t *mutex, int blocking)
 {
     unsigned irqstate = irq_disable();
-    DEBUG("%s: Mutex in use.\n", sched_active_thread->name);
+
+    DEBUG("PID[%" PRIkernel_pid "]: Mutex in use.\n", sched_active_pid);
 
     if (mutex->queue.next == NULL) {
         /* mutex is unlocked. */
         mutex->queue.next = MUTEX_LOCKED;
-        DEBUG("%s: mutex_wait early out.\n", sched_active_thread->name);
+        DEBUG("PID[%" PRIkernel_pid "]: mutex_wait early out.\n", sched_active_pid);
         irq_restore(irqstate);
         return 1;
     }
     else if (blocking) {
-        thread_t *me = (thread_t*) sched_active_thread;
-        DEBUG("%s: Adding node to mutex queue: prio: %" PRIu32 "\n", me->name, (uint32_t)me->priority);
+        thread_t *me = (thread_t *)sched_active_thread;
+        DEBUG("PID[%" PRIkernel_pid "]: Adding node to mutex queue: prio: %" PRIu32 "\n", sched_active_pid, (uint32_t)me->priority);
```

Comment:
if you fix the line length with this PR, you might as well fix it for all DEBUG statements?!